subworkflows/nf-core/initialise/main.nf

```diff
@@ -0,0 +1,55 @@
+// This is a reimplementation of the basic initialise methods in the nf-core template
+// The role of this subworkflow is to check the input parameters and print help messages
+// Use this to start your nf-core pipeline
+
+include { paramsHelp; paramsSummaryLog; validateParameters } from 'plugin/nf-validation'
+
+workflow INITIALISE {
+
+    if ( params.validate_parameters != false ){
+        validateParameters()
+    }
+
+    // Print citation for nf-core
+    def citation = "If you use ${workflow.manifest.name} for your analysis please cite:\n\n" +
+        "* The nf-core framework\n" +
+        "  https://doi.org/10.1038/s41587-020-0439-x\n\n" +
+        "* Software dependencies\n" +
+        "  https://github.com/${workflow.manifest.name}/blob/master/CITATIONS.md"
```

Comment:
This assumes that the pipeline is on github in that location, which is not always true for in-house pipelines developed using the nf-core tools but not designated for nf-core.